src/components/ViewdSeries.tsx

```diff
@@ -0,0 +1,79 @@
+import { BWListUser } from "@/hooks/useUserSeriesFetch";
+import { Card, Button } from "react-bootstrap";
+import "@/styles/MyPage.scss";
+import { useState } from "react";
+
+interface ViewdSeriesProps {
+  info: {
+    id: string;
+    name: string;
+    url: string;
+    seasons: number;
+    dateofnewseason: string;
+    bwlistsusers: BWListUser;
+  };
+  cbUpdateSeason: (id: string, numberSeason: number) => void;
+  cbHandleClickUpdate: () => void;
+  cbDeleteSeries: (id: string, paramReq: string) => void;
+}
+
+const ViewdSeries = ({
+  info,
+  cbUpdateSeason,
+  cbHandleClickUpdate,
+  cbDeleteSeries,
+}: ViewdSeriesProps) => {
+  const [numberSeason, setNumberSeason] = useState<number>(
+    info.bwlistsusers.numberofseason
+  );
+  const newSeasons =
+    info.dateofnewseason === "закрыт"
+      ? "Сериал закрыт"
+      : `Дата выхода ${info.seasons + 1} сезона - ${info.dateofnewseason}`;
+
+  const handleChangeSeason = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const numberOfSeason = Number(e.currentTarget.value);
+    setNumberSeason(numberOfSeason);
+    cbUpdateSeason(info.id, numberOfSeason);
```

Comment:
I prefer to use destructing. 

`const {id, seasons, dateofnewseason, ....}  = info`